ruma-events/CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Breaking changes:
   },
   sticker::StickerEventContent
   ```
+* Add `tag::TagName` type
```

Comment:
Adding a type is not a breaking change on its own. Maybe add "and use it for tag::Tags` or sth. like that.